backend/src/main/java/com/github/gadenko/gamereviewsite/backend/controller/GameReviewController.java

```diff
@@ -0,0 +1,34 @@
+package com.github.gadenko.gamereviewsite.backend.controller;
+
+import com.github.gadenko.gamereviewsite.backend.model.GameReview;
+import com.github.gadenko.gamereviewsite.backend.service.GameReviewService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/gamereview")
+public class GameReviewController {
+
+    private final GameReviewService gameReviewService;
+    @Autowired
+    public GameReviewController(GameReviewService gameReviewService) {
+        this.gameReviewService = gameReviewService;
+    }
+
+    @GetMapping
+    public List<GameReview> getGameReview (){
+        return gameReviewService.getgameReviewService();
+    }
+
+    @PostMapping
+    public GameReview postNewReview (@RequestBody GameReview newGameReview){
```

Comment:
Use a dto here, otherwise the user can choose the id and overwrite already existing GameReviews